### DISCUSSION TITLE
Is it possible to get rid of popups just for connection lost?

### DISCUSSION THREAD

**Discussion by wilhil** [dbeaver/dbeaver#11112] [Q&A]
Hi

I have several connections to databases that I can only connect to via different VPNs.

If I then disconnect from VPN or go to another, I get a popup every few seconds about the connection being lost.

Truthfully, other than this, I have never seen any other popup within DBeaver, so, I've seen the "Enable notification popup" option which I can disable, but, I'm not sure if there are other popups I may miss.

Is there anything I can do to just disable the warning on lost connection and nothing else?

Thanks

**Comment by Korkman** [comment]
I don't know how it was back when you posted this question, but now you have a list of notifications you can configure individually, including their popup and sound.

Window > Preferences > User Interface > Notifications

Click the small `[v]` in the table grid to uncheck unwelcome popups.
